_layouts/default.html

```diff
@@ -53,7 +51,202 @@
 
           {% include_cached footer.html %}
 
-
+    
```

Comment:
This move from `search.html` could be confusing from a contributing standpoint. Can you explain more the enhancement trade-off here?

Reply:
ok algolia js is back with the search html file